erc190/exceptions.py

```diff
@@ -3,3 +3,16 @@ class ValidationError(Exception):
     Error to signal something does not pass a validation check.
     """
     pass
+
+
+class NotImplementedError(Exception):
```

Comment:
I think you're overriding the built-in [`NotImplementedError`](https://docs.python.org/3/library/exceptions.html#NotImplementedError) here.  You should just use the built-in

Reply:
I didn't realize this was a built-in exception--I removed the custom one now.